website/deploy/docs/user-guide/explore-your-project-in-gcp/index.html

```diff
@@ -509,7 +509,7 @@ <h2 id="run-the-load-generator">Run the load generator</h2>
 
 	
 	
-	<div class="text-muted mt-5 pt-3 border-top">Last modified January 1, 0001
+	<div class="text-muted mt-5 pt-3 border-top">Last modified March 11, 2021: <a  href="https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/commit/111ec5a26b14ca30c615acd736507e915bc82ec6">Adding all of the new website code (111ec5a)</a>
```

Comment:
is there a way to not touch it manually and have it updated by a widget or similar?

Reply:
This is actually generated automatically when the website is generated. This will be automatically update/re-generated more often once we containerize or have Github Actions do the website generation.